core/app/finders/spree/taxons/find.rb

```diff
@@ -0,0 +1,61 @@
+module Spree
+  module Taxons
+    class Find
+      def initialize(taxons, params)
+        @taxons   = taxons
+
+        @roots    = params[:roots]
+        @ids      = params[:ids].split(',') if params[:ids].present?
```

Comment:
`Array(params[:ids]).split(',')` to avoid `if` ?

Reply:
actually, it's not a problem with transformation to Array but with calling `split. It'll fail if params[:ids] == nil

Reply:
`Array(nil) => []`

`Array([]) => []`